php/includes/ssp-functions.php

```
@@ -1722,6 +1719,8 @@ function ssp_get_episode_podcasts( $post_id ) {
 	 * @return WP_Term[]
 	 */
 	function ssp_get_podcasts( $hide_empty = false ) {
+		$cache_key = 'ssp_podcasts';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$cache_key'. (Unused Code Rules)

(UnusedLocalVariable)